src/components/search/Search.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+
+const architectsList = [
+  {
+    id: 1,
+    name: 'John',
+    surname: 'Doe',
+    city: 'Minsk',
+  },
+  {
+    id: 2,
+    name: 'Mike',
+    surname: 'Hike',
+    city: 'Hrodno',
+  },
+  {
+    id: 3,
+    name: 'Jane',
+    surname: 'Doe',
+    city: 'Vitebsk',
+  },
+  {
+    id: 4,
+    name: 'Sarah',
+    surname: 'Smith',
+    city: 'Homel',
+  },
+
+];
+
+class Search extends Component {
+  static searchFor(query) {
```

Comment:
I propose use trim function before start search 
```
const preparedQuery = query.trim().toLowerCase();
return artist => artist.name.toLowerCase().includes(preparedQuery)...
```
 and then there's this kind of thing